src/PeerConnectionManager.cpp

```diff
@@ -137,6 +141,35 @@ IceServer getIceServerFromUrl(const std::string &url, const std::string &clientI
 	return srv;
 }
 
+
+struct webrtcUdpPortRange
+{
+	int _minPort;
+	int _maxPort;
+};
+webrtcUdpPortRange setWebrtcUdpPortRange(const std::string &localWebrtcUdpPortRange)
+{
+	webrtcUdpPortRange portRange;
+	int pos = localWebrtcUdpPortRange.find(':');
+	if (pos == std::string::npos) {
+		RTC_LOG(INFO) << "Webrtc Udp port range is not defined";
+		portRange._minPort = 0;
+		portRange._maxPort = 65534;
+	}
+	else
+	{
+		std::string udpMin = localWebrtcUdpPortRange.substr(0, pos);
+		portRange._minPort = std::stoi(udpMin);
+		std::string udpMax = localWebrtcUdpPortRange.substr(pos + 1);
+		portRange._maxPort = std::stoi(udpMax);
+	}
+
+	return portRange;
+}
+webrtcUdpPortRange m_webrtcPortRange;
```

Comment:
This should be a member of PeerConnectionManager, using a global with a prefix "m_" confusing.